nxapi/nxlog.py

```diff
@@ -77,3 +126,75 @@ def explain_nxlog(nxlog):
 
     return explain
 
+def unify_date(date):
```

Comment:
Didn't know about python-dateutil, thanks for the suggestion. It is quite simplier, and more robust.